packages/OreIdWalletConnect/src/ConnectWalletContainer/ConnectWalletContainer.stories.tsx

```diff
@@ -34,7 +34,9 @@ Defaul.args = {
     chainNetwork: ChainNetwork.EthMain,
     account: '0x7FFBF659A640e181BA2Db633686Af123E9E1eE1b',
   },
-  createConnection: (...args) => console.log('createConnection', args),
+  createConnection: (walletConnectRef: WalletConnectRef) => {
+    console.log('createConnection: ', JSON.stringify(walletConnectRef.connector.session))
```

Comment:
please remove console.logs

Reply:
Tray, as it is in the storybook I think it's interesting to leave the `console.log`. It's interesting because it helps to demonstrate the component.